index.d.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * SpriteView utility
+ * @type {object}
+ * @author Berk Baski <berk.baski@smartface.io>
+ * @copyright Smartface 2021
+ */
+
+import ImageView from "sf-core/ui/imageview";
+import Image from "sf-core/ui/image";
+
+declare interface ISpriteViewOptions {
+    /**
+     * Width of the frame
+     */
+    width: number;
+    /**
+     * Height of the frame
+     */
+    height: number;
+    /**
+     * Fill type of the frame
+     */
+    imageFillType: ImageView.FillType;
+}
+
+export type SetSpriteOptions = {
+    /**
+     * Image for the frame
+     */
+    sheet: Image,
+    /**
+     * X Position of the window
+     */
+    frameX: number,
+    /**
+     * Y Position of the window
+     */
+    frameY: number,
+    /**
+     * Frame count of the image
+     */
+    frameCount: number
```

Comment:
Kill it!

Reply:
Make frameCount optional with default value to x*y

Reply:
Okay, did it!